tests/sanity/ignore-2.11.txt

```diff
@@ -1,2 +1,6 @@
 plugins/modules/ec2_tag.py validate-modules:parameter-state-invalid-choice  # Deprecated choice that can't be removed until 2022
 plugins/modules/ec2_vol.py validate-modules:parameter-state-invalid-choice  # Deprecated choice that can't be removed until 2022
+plugins/module_utils/cloud_py.py future-import-boilerplate!skip
+plugins/module_utils/cloud_py.py metaclass-boilerplate!skip
+tests/unit/module_utils/test_cloud.py future-import-boilerplate!skip
+tests/unit/module_utils/test_cloud.py metaclass-boilerplate!skip
```

Comment:
The CI tests already explicitly skip 'future-import-boilerplate' and 'metaclass-boilerplate', these lines shouldn't be needed.